### DISCUSSION TITLE
Markdown unordered-list lost bullets in docx

### DISCUSSION THREAD

**Discussion by mrkissinger** [jgm/pandoc#9857] [General]
I have 2 Linux laptops, both are running Debian 12.
I found pandoc on the old laptop runs correctly, but not on the new laptop.

I tested by this simple markdown file:

``` markdown
1. aaa
2. bbb
3. ccc

- 111
- 222
- 333
```

Then, generated docx file by:
$pandoc -o test.docx test.md

On the old laptop, the result seems correct:

![Screenshot from 2024-06-08 14-36-11](https://github.com/jgm/pandoc/assets/77801966/1f9700a1-17c9-447f-9c06-d96cffe55db9)

On the new laptop, the bullets are disappeared in docx:

![Screenshot from 2024-06-08 14-36-04](https://github.com/jgm/pandoc/assets/77801966/7e22ff0c-9972-4a35-8c5a-5b061c4f93fe)

I checked the packages version, they are all the same.

Does anyone  know what may cause this problem?

**Comment by tarleb** [comment]
Can you try with the latest version? You can get it [here](https://github.com/jgm/pandoc/releases).

**Reply by mrkissinger** [reply]
I tried the lastest version, but the problem is still there.

**Reply by tarleb** [reply]
My guess is that you're missing a font, so the bullets don't render correctly. Please try to view the working file on the other computer and see if things look identical.

**Reply by mrkissinger** [reply]
I generated the docx file on the new laptop, and view it on the old laptop, the bullets are still disappeared.
At the same time, the docx file that generated on the old laptop, the bullets are OK.

The attachment contains 2 docx files.
1.docx is generated on the new laptop.
2.docx is generated on the old laptop.

[docx.tar.gz](https://github.com/user-attachments/files/15747426/docx.tar.gz)

**Reply by mrkissinger** [reply]
I upgraded pandoc to v3.2 on my old laptop, the same problem appeared on the old laptop.

Downgrade to v3.1.3+ds-3 by Debian apt, it works again.

**Reply by tarleb** [reply]
Comparing the zip contents of those files reveals many differences. You're probably using different "reference-doc" values. Check for a `reference.docx` file in your user data directory (see `pandoc -v`). It could also be that you have different versions of the `pandoc-data` package installed.

**Reply by mrkissinger** [reply]
Not reference-doc problem. I used pandoc without reference-doc, the problem is still there.
But I downgrade pandoc to v3.1.3+ds-3 by Debian apt on my new laptop, it works now.

**Comment by jgm** [comment]
There were some significant changes in docx rendering of unordered lists in the recent versions. The intent was to make pandoc's output similar to default Word output.  
See commit b6b668ee3f3ac7782154171f7a078d5857d97dfc

Here's a guess:  are you using LibreOffice to view the docx?

Pandoc now uses characters from standard Word fonts (Symbol, Courier New, Wingdings) for bullets, to match Word's default behavior. Perhaps LibreOffice lacks these?  (For what it's worth, LibreOffice on macOS renders these with non problem.)

**Reply by tarleb** [reply]
Ah thanks, I wasn't aware. An archive with both zip files is linked in [this comment](https://github.com/jgm/pandoc/discussions/9857#discussioncomment-9712181).

**Reply by jgm** [reply]
OK.  I can report that both 1.docx and 2.docx display properly for me on macOS, both with Word and with LibreOffice.

Most likely this is a font issue of some kind, but it would be good to diagnose further.

**Reply by mrkissinger** [reply]
I am using WPS (https://linux.wps.cn/) as MS Office alternative.
It works on pandoc v3.1.3+ds-3, but not later version.

I tried LibreOffice, seems it worked.

The ttf-mscorefonts-installer package is installed.

Is it possible to custom the bullet font in reference-doc?

**Comment by jgm** [comment]
Can you upload the generated docx so we can take  a look?

**Comment by jgm** [comment]
Here's something to try: install `ttf-mscorefonts-installer` package (and presumably run it to install the fonts?).
See if that fixes it.

**Reply by mrkissinger** [reply]
I installed ttf-mscorefonts-installer on both laptops, the problem is still there.

**Reply by jgm** [reply]
Can you verify that the MS fonts are correctly installed?  E.g. if you open Word or LibreOffice, do you see Symbol, Courier New, and Wingdings as options?

**Reply by jgm** [reply]
It's possible this is an issue with your word processor software. Have you tried using LibreOffice?
